en/airframes/adding_a_new_frame.md

```diff
@@ -115,12 +117,14 @@ S: 0 1   6500   6500      0 -10000  10000
 
 Where each number from left to right means:
 
-* M: Indicates two scalers for two inputs
-* O: Indicates the output scaling (*1 in negative, *1 in positive), offset (zero here), and output range (-1..+1 here).  If you want to invert your PWM signal, the signs of the output scalings has to be changed. (```O:      -10000  -10000      0 -10000  10000```)
-* S: Indicates the first input scaler: It takes input from control group #0 (attitude controls) and the first input (roll). It scales the input * 0.6 and reverts the sign (-0.6 becomes -6000 in scaled units). It applies no offset (0) and outputs to the full range (-1..+1)
-* S: Indicates the second input scaler: It takes input from control group #0 (attitude controls) and the second input (pitch). It scales the input * 0.65 and reverts the sign (-0.65 becomes -6500 in scaled units). It applies no offset (0) and outputs to the full range (-1..+1)
+* M: Indicates two scalers for two control inputs. It indicates the number of control inputs the mixer will receive.
+* O: Indicates the output scaling (*1 in negative, *1 in positive), offset (zero here), and output range (-1..+1 here).  If you want to invert your PWM signal, the signs of the output scalings have to be changed. (```O:      -10000  -10000      0 -10000  10000```)
+* S: Indicates the first input scaler: It takes input from control group #0 (Flight Control) and the first input (roll). It scales the roll control input * 0.6 and reverts the sign (-0.6 becomes -6000 in scaled units). It applies no offset (0) and outputs to the full range (-1..+1)
+* S: Indicates the second input scaler: It takes input from control group #0 (Flight Control) and the second input (pitch). It scales the pitch control input * 0.65. It applies no offset (0) and outputs to the full range (-1..+1)
```

Comment:
This line is not consistent with the example above in which the sign is not reverted, so either change this line or the example above.
The error is most likely due to choosing the wrong block of code from the complete mixer. The first block of code (for Servo 0 was chosen) but the explanation is only correct for the second block of code (for Servo 1).

Reply:
@barzanisar Can you check this

Reply:
@RicardoM17  line 123 doesn't say the sign is reverted. Previously (line 121) it said it reverts the sign and this is why I changed it. I don't understand why line 123 is still not consistent with the example given. Can you please explain?

Reply:
@barzanisar maybe it was already fixed but in the documentation ( see here: https://dev.px4.io/en/airframes/adding_a_new_frame.html) it still says revert the sign. But you are correct that line 123 here doesn't say it reverts the sign. However line 123 here is not consistent with the one in the link I sent you. I was discussing this with @hamishwillee and when I commented here I didn't re-read everything (I assumed it was the same) so maybe it's already been fixed.

Reply:
Sounds like there is no problem them. Thanks guys for working on this.